standup/settings.py

```diff
@@ -124,6 +124,7 @@ def path(*paths):
         },
     },
 ]
+DEFAULT_JINJA2_TEMPLATE_EXTENSION = '.html'
```

Comment:
I found that in order to use these convenience error views you need to set this to the extension of your error templates unless you use the default: `.jinja`. See [the code](https://github.com/niwinz/django-jinja/blob/master/django_jinja/views/__init__.py#L51).